pyFAI/azimuthalIntegrator.py

```diff
@@ -1202,7 +1190,7 @@ def integrate1d_ng(self, data, npt, filename=None,
                 variance = (numpy.maximum(data, 1.0) + numpy.maximum(dark, 0.0)).astype(numpy.float32)
 
         # Prepare LUT if needed!
-        if method.algo_lower in ("csr", "lut"):
+        if method.algo_lower in ("csr", "lut", "csc"):
```

Comment:
Could this kind of decision be inside the 'method' object?

Reply:
Yes it could, it is even foreseen to be made like this but it will requires a major refactoring of all the `ai.integrate1d_ng` method.